web/src/routes/(user)/people/+page.svelte

```diff
@@ -45,14 +52,31 @@
   let potentialMergePeople: PersonResponseDto[] = [];
   let edittingPerson: PersonResponseDto | null = null;
 
-  people.forEach((person: PersonResponseDto) => {
+  let peopleCopy = data.people.people;
+
+  for (const person of peopleCopy) {
     initialHiddenValues[person.id] = person.isHidden;
-  });
+  }
+
+  $: {
+    if (searchName) {
+      people = searchNameLocal(searchName, searchPeopleCopy, 10);
```

Comment:
```suggestion
      people = searchNameLocal(searchName, searchPeopleCopy, 100);
```
The result set is limited to 10. Can you raise that to...let's say 100?
I think this is more appropriate for the big people list.

Reply:
I increased it to 20; the API to search people returns only 20 people